src/index.css

```diff
@@ -8,7 +8,7 @@
   font-family: "DM Sans", sans-serif;
   line-height: 1.5;
   font-weight: 400;
-  color-scheme: light dark;
+  color-scheme: dark light;
```

Comment:
Not sure if that makes a difference,  I was testing the code and it came out that way